rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -58,17 +56,6 @@ class FederationSupportImplTest {
     private final FederationSupportBuilder federationSupportBuilder = new FederationSupportBuilder();
     private FederationSupport federationSupport;
 
-    @BeforeEach
-    void setUp() {
-        StorageAccessor storageAccessor = new InMemoryStorage();
-        storageProvider = new FederationStorageProviderImpl(storageAccessor);
-
-        federationSupport = federationSupportBuilder
-            .withFederationConstants(federationMainnetConstants)
-            .withFederationStorageProvider(storageProvider)
-            .build();
-    }
-
```

Comment:
this can be deleted now since the test that was using it has been refactored and placed into the inner classes